packages/teams-js/src/public/pages.ts

```diff
@@ -1096,4 +1103,117 @@ export namespace pages {
         : false;
     }
   }
+
+  /**
+   * Provides APIs to interact with the button in hosts
+   *
+   * @beta
+   */
+  export namespace responseButton {
+    /**
+     * The required information that the app needs to let host apps know.
+     *
+     * @beta
+     */
+    export interface ResponseInfo {
+      /**
+       * The unique ID helps the host and apps identify which button it refers to.
+       *
+       * @beta
+       */
+      responseId: string;
+      /**
+       * Id of File on OneDrive
+       *
+       * @beta
+       */
+      oneDriveFileId: string;
+      /**
+       * The exact message where the action was invoked from
+       *
+       * @beta
+       */
+      messageId: string;
+
+      /**
+       * The conversation where the action was invoked from
+       *
+       * @beta
+       */
+      conversionId: string;
+    }
+
+    /**
+     * Notify host apps to show the response button.
+     * @param params - Parameters sent to host apps contain the required information.
+     *
```

Comment:
This should also have a `@returns` section in the comment talking about how it returns a `Promise<void>` that will resolve when the ResponseButton is visible in the host.